drops/azure_edge_extensions_rag_on_edge_multimodal.json

```diff
@@ -0,0 +1,27 @@
+{
+  "Title": "Multi-Modal Retrieval-Augmented Generation on Edge with Azure Arc & Azure IoT Operations",
+  "Summary": "This Drop provides a multi-modal (image + text) Retrieval-Augmented Generation(RAG) on edge pattern that facilitates facilitates multi-modal text and image searches within files utilizing a multi-modal vector search engine Marqo and an multi-modal Large Language Model LLaVa.",

```

Comment:
Maybe this is a bit more clear: 

This Drop provides a multi-modal (image + text) Retrieval-Augmented Generation (RAG) on edge pattern that facilitates multi-modal text and image searches within files, utilizing a multi-modal vector search engine, Marqo, and a multi-modal Large Language Model, LLaVa

Reply:
updated